peak/peak.py

```diff
@@ -1,2 +1,45 @@
+from collections import OrderedDict
+from bit_vector import BitVector
+
 class Peak:
     pass
+
+
+def name_outputs(**outputs):
+    """Decorator meant to apply to any function to specify output types
+    The output types will be stored in fn.__peak_outputs__
+    The input types will be stored in fn.__peak_inputs__
+    Will verify that all the inputs have type annotations
+    Will also verify that the outputs of running fn will have the correct number of bits
+    """
+    def decorator(call_fn):
+        def call_wrapper(*args,**kwargs):
```

Comment:
should decorate `call_wapper` with `@functools.wraps(call_fn)`
https://docs.python.org/3/library/functools.html#functools.wraps